core/src/test/java/org/jboss/pnc/causeway/ctl/ImportControllerTest.java

```diff
@@ -86,8 +87,15 @@
     private static final String BUILD_VERSION = "1.1.1";
     private static final String ARTIFACTS_VERSION = "1.1.1.redhat_1";
     private static final String BUILD_NAME = "org.apache.geronimo.specs:geronimo-annotation_1.0_spec";
-    private static final String CALLBACK_URL = "http://localhost:8081/callback";
-    private static final CallbackTarget CALLBACK_TARGET = new CallbackTarget(CALLBACK_URL, Request.Method.POST);
+    private static final URI CALLBACK_URL;
+    static {
+        try {
+            CALLBACK_URL = new URI("http://localhost:8081/callback");
```

Comment:
`URI.create(String str)`, does basically the same thing without the need for a static block

Reply:
Yup, I've learned that yesterday. I'll update it here.